tests/d3d12.c

```diff
@@ -42353,6 +42353,7 @@ static void test_get_resource_tiling(void)
         UINT tile_shape_h;
         UINT tile_shape_d;
         D3D12_TILED_RESOURCES_TIER min_tier;
+        bool todo_amd;
```

Comment:
Is this TODO on our end, or bug in RADV? Should probably be called todo_radv in that case.